### DISCUSSION TITLE
Telemundo Channels + Gray Television & NBC Channels

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#1192] [Show and tell]
I Was Trying To Find Any NBC Telemundo m3u8s That Worked, Well I Did Find Few Working m3u8s but They Don't Show The Telemundo Signal But Only The News Segment Of The Channel And Some of m3u8s Sometimes Show NBC. Some Of The Links Start With nbculocallive.akamaized And Some Are From Gray Television.

So Here Every m3u8s From nbculocallive.akamaized 

Telemundo Área De La Bahía
https://nbculocallive.akamaized.net/hls/live/2037097/bayarea/stream3/master.m3u8

Telemundo Chicago
https://nbculocallive.akamaized.net/hls/live/2037085/chicago/stream4/master.m3u8

Telemundo Dallas Fort-Worth TX
https://nbculocallive.akamaized.net/hls/live/2037095/dallas/stream1/master.m3u8

Telemundo Denver CO
https://nbculocallive.akamaized.net/hls/live/2037092/denver/stream1/master.m3u8

Telemundo El Paso TX
https://nbculocallive.akamaized.net/hls/live/2037500/elpaso/stream1/master.m3u8

Telemundo Fresno CA
https://nbculocallive.akamaized.net/hls/live/2037601/fresno/stream1/master.m3u8

Telemundo Houston TX
https://nbculocallive.akamaized.net/hls/live/2037088/houston/stream1/master.m3u8

Telemundo Filadelfia / [Philadelphia] https://nbculocallive.akamaized.net/hls/live/2035366/philadelphia/stream5/master.m3u8

Telemundo Las Vegas NV
https://nbculocallive.akamaized.net/hls/live/2037093/lasvegas/stream1/master.m3u8

Telemundo Los Angeles CA
https://nbculocallive.akamaized.net/hls/live/2037084/losangeles/stream6/master.m3u8

Telemundo McAllen TX
https://nbculocallive.akamaized.net/hls/live/2037094/mcallen/stream1/master.m3u8

Telemundo Miami FL
https://nbculocallive.akamaized.net/hls/live/2037498/miami/stream6/master.m3u8

Telemundo New York
https://nbculocallive.akamaized.net/hls/live/2037083/newyork/stream7/master.m3u8

Telemundo Nuevo México
https://nbculocallive.akamaized.net/hls/live/2037500/elpaso/stream2/master.m3u8

Telemundo Orlando FL
https://nbculocallive.akamaized.net/hls/live/2037602/orlando/stream1/master.m3u8

Telemundo Phoenix AZ
https://nbculocallive.akamaized.net/hls/live/2037089/phoenix/stream1/master.m3u8

Telemundo Puerto Rico (WKAQ)
https://nbculocallive.akamaized.net/hls/live/2037499/puertorico/stream1/master.m3u8

Telemundo Sacramento CA
https://nbculocallive.akamaized.net/hls/live/2037604/sacramento/stream1/master.m3u8

Telemundo Salt Lake City UT
https://nbculocallive.akamaized.net/hls/live/2037605/saltlakecity/stream1/master.m3u8

Telemundo San Antonio TX
https://nbculocallive.akamaized.net/hls/live/2037091/sanantonio/stream1/master.m3u8

Telemundo San Diego CA
https://nbculocallive.akamaized.net/hls/live/2037098/sandiego/stream3/master.m3u8

Telemundo Tampa FL
https://nbculocallive.akamaized.net/hls/live/2037606/tampa/stream1/master.m3u8

Telemundo Washington DC
https://nbculocallive.akamaized.net/hls/live/2037099/washingtondc/stream4/master.m3u8

---------------------
[NBC]
NBC 11 Bay Area
https://nbculocallive.akamaized.net/hls/live/2037097/bayarea/stream4/master.m3u8

NBC 5 Chicago
https://nbculocallive.akamaized.net/hls/live/2037085/chicago/stream1/master.m3u8

NBC 5 Dallas Fort-Worth
https://nbculocallive.akamaized.net/hls/live/2037095/dallas/stream5/master.m3u8

NBC 4 Los Angeles
https://nbculocallive.akamaized.net/hls/live/2037084/losangeles/stream1/master.m3u8

NBC 4 New York
https://nbculocallive.akamaized.net/hls/live/2037083/newyork/stream1/master.m3u8

NBC 7 San Diego
https://nbculocallive.akamaized.net/hls/live/2037098/sandiego/stream1/master.m3u8

NBC 6 South Florida [Miami FL]
https://nbculocallive.akamaized.net/hls/live/2037498/miami/stream1/master.m3u8

NBC 10 Philadelphia
https://nbculocallive.akamaized.net/hls/live/2035366/philadelphia/stream2/master.m3u8

NBC 4 Washington DC
https://nbculocallive.akamaized.net/hls/live/2037099/washingtondc/stream1/master.m3u8

And Here The Links From Gray Television

Telemundo Atlanta GA
https://dai2-playlistserver.aws.syncbak.com/cpl/20475903/dai2v5/1.0/7b2264657669636554797065223a387d/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IkdyYXkyMDE2MDgyOSJ9.eyJtaWQiOiIyMDQ3NTkwMyIsImNpZCI6MjA5NzQsInNpZCI6MTU4LCJtZDUiOiJiMTgwM2ExYTIxMzA5ZTEwNzNlOGJjMDQyOGU5NDkzOCIsImlhdCI6MTY1Mjg5MjgyMSwiZXhwIjoxNjg0NDI4ODIxLCJpc3MiOiJTeW5jYmFrIChURykifQ.ngWlxw_iUrju6oMpQl73BYHMCl7c6reypSqqP5nLbL8

Telemundo Laredo TX
https://dai2-playlistserver.aws.syncbak.com/playlist/13355198/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsImtpZCI6IkdyYXkyMDE2MDgyOSIsInN1YiI6IioiLCJ0eXAiOiJKV1QifQ.eyJleHAiOjE3OTE2NDc1MDIsImlzcyI6IldMUyIsIm1kNSI6IjQwMWRkMDY2YjUyOGEwZWIxYzcxMzMxNjBhNDgyMGE5IiwibWlkIjoiMTMzNTUxOTgifQ.piLwptYMsquOvh5kVgvpuj2wYSd6SDOblXdfIXGk7PE

Telemundo 20 Odessa TX
https://dai2-playlistserver.aws.syncbak.com/playlist/19106378/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IkdyYXkyMDE2MDgyOSJ9.eyJtaWQiOiIxOTEwNjM3OCIsImNpZCI6MTgwMjMsInNpZCI6MTU4LCJtZDUiOiIyMmMxY2M4MTVmMmY1YzJmMjZhZDEzYmQ2MTI1NjFhYSIsImlhdCI6MTY0NDM1Njk4MSwiZXhwIjoxNjc1ODkyOTgxLCJpc3MiOiJTeW5jYmFrIChURykifQ.FluzuQn2DLfJeSKreT0h2OHKpoBGNSkmeu7RKb3O2h0

---------------------
[NBC]
NBC 3 Jackson MS (WLBT)
https://dai2-playlistserver.aws.syncbak.com/cpl/20965226/dai2v5/1.0/7b2264657669636554797065223a387d/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IkdyYXkyMDE2MDgyOSJ9.eyJtaWQiOiIyMDk2NTIyNiIsImNpZCI6MjE4ODgsInNpZCI6MTU4LCJtZDUiOiJjYTU5Nzc4YjUxZjZkNTZjODFmOWZiZjJjZmQzNDNiMiIsImlhdCI6MTY2MDkyOTM0NywiZXhwIjoxNjkyNDY1MzQ3LCJpc3MiOiJTeW5jYmFrIChURykifQ.5heNHIgDbrnFzfBjH4qmNs3sFGUl1iewhcB-Iru-4ok

NBC 7 Lake Charles LA (KPLC)
https://dai2-playlistserver.aws.syncbak.com/cpl/20979933/dai2v5/1.0/7b2264657669636554797065223a387d/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IkdyYXkyMDE2MDgyOSJ9.eyJtaWQiOiIyMDk3OTkzMyIsImNpZCI6MjE5MDAsInNpZCI6MTU4LCJtZDUiOiJmNDhkOTMzZGZiNWQyY2RjNWU1NTcyNjQ3MmQ1YjFlNSIsImlhdCI6MTY2MTQ0NjY2OCwiZXhwIjoxNjkyOTgyNjY4LCJpc3MiOiJTeW5jYmFrIChURykifQ.C51yBQc-ZSAkhWe0zCXWP3tUJ8MzijUNOaes6Zddk0w

**Comment by AntiPontifex** [comment]
Amazing work!

**Comment by interactyba** [comment]
Can you obtain more NBC/Telemundo Live links?
I was trying with "telemundolive.akamaized.net", but I did not got nothing.
Thank you!

**Reply by R4IN2007** [reply]
I will try and see if I can find any more NBC/Telemundo Links.

**Reply by interactyba** [reply]
👍

**Reply by R4IN2007** [reply]
Here Few m3u8s that I found
*TELEMUNDO LINKS*

Telemundo 3 Yuma AZ (KESE-LD)
https://content.uplynk.com/channel/d8ebf84aa99d430980a7995d658ece0a.m3u8

Telemundo Wisconsin (WDJT/WYTU-LD) [NOT 24/7]
https://video.cbs58.com:443/live/wdjt/live4/live4_720p.m3u8

*NBC LINKS*

NBC 13 Birmingham AL (WVTM)
https://content.uplynk.com/channel/8e7504962da04956953c874a7ad11c6b.m3u8

NBC 12 Mesa AZ (KPNX)
https://livevideo.tegnadigital.com/kpnx/v1/master/f9c1bf9ffd6ac86b6173a7c169ff6e3f4efbd693/KPNX/elvs/live.m3u8

NBC 6 San Luis Obispo CA (KSBY)
https://content.uplynk.com/channel/5545aa8a7e5a480d85fd1cff8d0cf610.m3u8

NBC 9 Denver CO (KUSA)
https://livevideo.tegnadigital.com/kusa/v1/master/f9c1bf9ffd6ac86b6173a7c169ff6e3f4efbd693/KUSA/elvs/live.m3u8

NBC 5 West Palm Beach FL (WPTV)
https://content.uplynk.com/channel/b02350ef854c4eef8358eead1f59a5cc.m3u8

NBC 2 Fort Myers FL (WBBH)
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/watch?v=MJB6CUxvZcU

NBC 11 Atlanta GA (WXIA)
https://livevideo.tegnadigital.com/wxia/v1/master/f9c1bf9ffd6ac86b6173a7c169ff6e3f4efbd693/WXIA/elvs/live.m3u8

NBC 7 Boise ID (KTVB)
https://livevideo.tegnadigital.com/ktvb/v1/master/f9c1bf9ffd6ac86b6173a7c169ff6e3f4efbd693/KTVB/elvs/live.m3u8

**Reply by interactyba** [reply]
Amazing! I thank you for that! If you know either where I can find channels live links, or if you can find any more, I will be a lot of thank.

**Reply by 3v3rm4rt3lL** [reply]
Hello, any Unimas  west or east, links? Thanks.

**Comment by BellezaEmporium** [comment]
Amazing @R4IN2007 !

**Comment by democratia1987-source** [comment]
Hello. Are these channels blocked locally?
